test/unit/signal.spec.js

```diff
@@ -1,107 +1,152 @@
-import Signal from '../../lib/api/signal'
+import { MemoizedSignal, Signal } from '../../lib/api/signal'
 import {noop} from '../helpers'
 
-describe(`Signal`, () => {
-  describe(`constructor`, () => {
-    it(`creates a Signal instance`, () => {
-      expect(new Signal()).to.be.instanceof(Signal)
-    })
-  })
 
-  describe(`instance`, () => {
-    let signal
+function runCommonSpecs (SignalConstructor, initialValue = 42) {
```

Comment:
Please rename it to something meaningful like `behavesLikeSignal`.

Also the `initialValue` is never set as a prameter, so please remove it.